manager/controllers/utils/configuration.go

```diff
@@ -15,9 +15,16 @@ import (
 const (
 	CatalogConnectorServiceAddressKey string = "CATALOG_CONNECTOR_URL"
 	VaultAddressKey                   string = "VAULT_ADDRESS"
-	VaultSecretKey                    string = "VAULT_TOKEN"
-	VaultTTLKey                       string = "VAULT_TTL"
-	VaultModulesRole                  string = "VAULT_MODULES_ROLE"
+	// FIXME: Current communication between the manager and Vault assumes a Vault
+	// token which is passed as an env var.
+	// This should be changed to the following:
+	// 1. get a Vault token form Vault using the manager role
+	// 2. continue communication with Vault using this Vault token
+	// The manager role should be passed as env var after it was configured
+	// with the appropriate permissions for the manager.
+	VaultSecretKey   string = "VAULT_TOKEN"
+	VaultTTLKey      string = "VAULT_TTL"
```

Comment:
We are not using neither VAULT_TOKEN nor VAULT_TTL, so why keep it?

Reply:
Not sure what is the purpose of this PR. 
1. manager role env is not introduced, only mentioned in a comment
2. If vault token should be obtained by the manager, why is it still in the configmap and configuration.go file?